src/main/java/ch/njol/skript/classes/data/BukkitEventValues.java

```diff
@@ -1380,6 +1381,16 @@ public AbstractVillager get(PlayerTradeEvent event) {
 				}
 			}, EventValues.TIME_NOW);
 		}
+		// EntityJumpEvent
```

Comment:
Is this needed? I believe there is an entity event value implemented for EntityEvent itself

Reply:
Hmm... As far as I can see, I'm not entirely sure.

I mean, there is an [isJumping()](https://jd.papermc.io/paper/1.16/org/bukkit/entity/LivingEntity.html#isJumping()) for Paper in LivingEntity, but other than that, there's not really anything useful that I can find.

Reply:
I just mean that I don't think you need to add this event value. event-entity should be usable without it because https://github.com/SkriptLang/Skript/blob/69ef6de6adf1e7dbfb76a1dbb830d955c04ef382/src/main/java/ch/njol/skript/classes/data/BukkitEventValues.java#L401 exists :D

Reply:
Oh so that's what you meant xD

I'll give it a test soon and then commit the changes. Thank you for letting me know :D

Reply:
Update!

- [Merge branch 'master' into EntityJumpEvent](https://github.com/SkriptLang/Skript/pull/5123/commits/8096b328cc51b5ce2c5b2b95730375cf343f66b3)
- [Added @APickledWalrus's review/removed event-value](https://github.com/SkriptLang/Skript/pull/5123/commits/51fb2a9f2dc2c9fb0cd43495a3a4f9c1bc47be50)

Thank you for your review :D